packages/tupaia-web/src/layout/UserMenu/PopoverMenu.tsx

```diff
@@ -47,7 +49,20 @@ export const PopoverMenu = ({
         horizontal: 'right',
       }}
     >
-      <MenuList secondaryColor={secondaryColor}>{children}</MenuList>
+      <MenuList secondaryColor={secondaryColor}>
+        {/** If the user is not logged in, show the register and login buttons */}
+        {!isUserLoggedIn && (
+          <>
+            <MenuItem href="/login" onCloseMenu={onCloseMenu}>
+              Log in
+            </MenuItem>
+            <MenuItem href="/register" onCloseMenu={onCloseMenu}>
+              Register
+            </MenuItem>
+          </>
+        )}
+        {children}
+      </MenuList>
```

Comment:
Are these going to be in the popover menu? Or still in the top bar?

Reply:
I have moved this to the top bar